### ISSUE TITLE
High information density/"ultra compact" view

### ISSUE THREAD

**Issue by yukichigai** [MbinOrg/mbin#338]
While Mbin has numerous options for adjusting and optimizing page layout, even its most compact view is lacking in information density compared to what is possible on other fediverse platforms. For example, using the most compact settings on Mbin will display literally half the number of topics on the front page as are visible on a Lemmy instance using the MLMYM (old reddit style) frontend. Comments are similarly affected.

An additional "ultra compact" view offering higher information density would be welcome. This shouldn't require much other than significantly reducing the white space and other padding in display elements.

(For the record, I have attempted to rectify this myself via custom userstyles, but I am... shall we say _not the best_ when it comes to modifying frontend presentation)

**Comment by e-five256** [comment]
I think I might need more information on this one:

> Mbin will display literally half the number of topics on the front page as are visible on a Lemmy instance using the MLMYM (old reddit style) 

I'm not able to observe this. Mbin shows 25 entries per page with infinite scroll disabled, Lemmy by default appears to show 20, and MLMYM shows 25, so matches Mbin

Is your issue related to the display of the entries themselves, or the number? There is also infinite scroll option in the sidebar to view more pages without having to click

**Comment by yukichigai** [comment]
Ah, my use of "page" was confusing, fair enough. I'm talking about the number of entries displayed on a single screen without scrolling. Also this is on desktop.

To get more specific, my secondary monitor has a vertical resolution of 1200 pixels. If I view Mbin on a browser window on this monitor using the smallest text settings and the "compact view" option I can see 8 threads on the screen at a time. While I can scroll to see more, I am still limited to only seeing 8 at a time. Comparatively, if I view a Lemmy instance via MLMYM I will see 16 on the screen at a time. Far more information displayed before I need to scroll, basically.

In other words, it's the amount of vertical pixel space each thread takes up. Mbin uses a lot of vertical space even using its most compact settings, ~115 vertical pixels per thread on Mbin vs ~65 per on MLMYM.

**Comment by yukichigai** [comment]
Still hoping for some attention on this.

**Comment by melroy89** [comment]
> Still hoping for some attention on this.

Labelled as good first issue.
